appinventor/appengine/war/index.jsp

```diff
@@ -62,8 +64,17 @@
         <li> Firefox 52+ </li>
       </ul>
     </div>
+    <!-- Message to display if ode doesn't load. When Ode loads,
+         this message will be hidden. -->
+    <div id=odeblock>
+        <h1>If you see this message for an extended period of time, it might be because
+        your internet service is blocking requiests to <%= odeBase %>. Contact your
+        administrator to check on this and remove the block.
+        </h1>
+    </div>
```

Comment:
We may only want to show this if `!odeBase.isEmpty()`. We should be able to test by loading localhost:8888 with Javascript turned off.